inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/service/RecommendationServiceImpl.java

```diff
@@ -358,5 +372,40 @@ public void putTrainedModel(Recommender aRecommender, Object aModel)
         {
             trainedModels.put(aRecommender.getId(), aModel);
         }
+
+        public void removePredictions(Recommender aRecommender, Project aProject)
```

Comment:
The recommender has a reference to the project. No need to pass the project separately.